src/containers/Transactions/test/Meta.test.tsx

```diff
@@ -32,35 +32,35 @@ describe('TransactionMeta container', () => {
     expect(w.find('li').length).toBe(23)
 
     expect(w.find('li').at(0).html()).toBe(
-      '<li class="meta-line">owned_account_root <a class="account" title="rUmustd4TbkjaEuS7S1damozpBEREgRz9z" href="/accounts/rUmustd4TbkjaEuS7S1damozpBEREgRz9z">rUmustd4TbkjaEuS7S1damozpBEREgRz9z</a><ul class="meta-line"><li>Balance decreased by<b>-\uE90050.324316<small>XRP</small></b>from<b>\uE9002,910.704988<small>XRP</small></b>to<b>\uE9002,860.380672<small>XRP</small></b></li></ul></li>',
+      '<li class="meta-line">owned_account_root <a title="rUmustd4TbkjaEuS7S1damozpBEREgRz9z" class="account" href="/accounts/rUmustd4TbkjaEuS7S1damozpBEREgRz9z">rUmustd4TbkjaEuS7S1damozpBEREgRz9z</a><ul class="meta-line"><li>Balance decreased by<b>-\uE90050.324316<small>XRP</small></b>from<b>\uE9002,910.704988<small>XRP</small></b>to<b>\uE9002,860.380672<small>XRP</small></b></li></ul></li>',
```

Comment:
why did the title and class change order?

Reply:
`react-router` changed them for `<Link>`'s output.

Reply:
It is referenced in "Context of Change"